### DISCUSSION TITLE
Merge Mylyn Docs Repository into Mylyn Repository

### DISCUSSION THREAD

**Discussion by BeckerFrank** [eclipse-mylyn/.github#35] [Announcements]
We have integrated the Mylyn Docs Repository into the Mylyn Repository.

The first Commit was [81e1520](https://github.com/eclipse-mylyn/org.eclipse.mylyn/commit/81e152045e4546567679f47d623c9fea41027063). So now short after Release of Mylyn 4.1.0 we want to archive Mylyn Docs Repository and have all Sources in the Mylyn Repository.

Further details can be found [here](https://github.com/eclipse-mylyn/org.eclipse.mylyn/issues/315)

Help in completing the open tasks is welcome.

**Comment by olberger** [comment]
FWIW, this probably explains the " This repository has been archived by the owner on Dec 5, 2023. It is now read-only." mention on the repo at https://github.com/eclipse-mylyn/org.eclipse.mylyn.docs ... but without any redirection there :-/

Maybe it could be a good idea to add some kind of redirection message if it's not completely read-only yet
